index.js

```diff
@@ -6,13 +6,19 @@ module.exports = {
 
   setupPreprocessorRegistry(type, registry) {
     if (type === 'parent') {
-      let HbsMinifierPlugin = require('./hbs-minifier-plugin');
-
+      let config;
+      try {
+        let path = this.isDevelopingAddon() ? `${this.project.root}/tests/dummy` : this.project.root;
+        config = require(`${path}/.hbs-minifyrc.js`);
+      } catch (error) {
+        throw error;
+      }
```

Comment:
Is it ok to have the config in a separate file?

Reply:
I'd prefer it if we used a `ember-hbs-minifier` key in `ember-cli-build.js` instead

Reply:
Since both app and add-on could make use of this minifier, having a separate config file would enable us to specify different configurations.

Reply:
The same should be possible if it is part of the usual `options` hash like in `ember-cli-babel`

Reply:
Alright. I'll make the changes.

Reply:
I have made the changes